src/components/Calendar/src/Calendar.vue

```diff
@@ -121,214 +109,180 @@ export default {
 
 	props: {
 		/**
-		 * Language specific format to use
-		 */
-		locale: {
-			type: String,
-			default: 'en-US',
-		},
-		/**
-		 * Selected date value (YYYY-MM-DD)
+		 * Selected date value in ISO format.
 		 */
 		selectedDate: {
 			type: String,
-			default: '',
-			validator: isDateFormat,
+			default: undefined,
+			validator: isIsoFormat,
 		},
 		/**
-		 * Disable the dates before the min-date
+		 * Disable the dates before the min-date in ISO format.
 		 */
 		minDate: {
 			type: String,
-			default: '',
-			validator: isDateFormat,
+			default: undefined,
+			validator: isIsoFormat,
 		},
 		/**
-		 * Disable the dates after the max-date
+		 * Disable the dates after the max-date in ISO format.
 		 */
 		maxDate: {
 			type: String,
-			default: '',
-			validator: isDateFormat,
+			default: undefined,
+			validator: isIsoFormat,
 		},
 		/**
-		 * List of disabled dates
+		 * List of disabled dates in ISO format.
 		 */
 		disabledDates: {
 			type: Array,
 			default: () => ([]),
-			validator: (v) => v.every((date) => isDateFormat(date)),
+			validator: (disabledDates) => disabledDates.every((date) => isIsoFormat(date)),
+		},
+		/**
+		 * Calendar locale. Defaults to browser locale.
+		 */
+		locale: {
+			type: String,
+			default: undefined,
 		},
 	},
 
 	data() {
 		return {
-			viewingDate: this.selectedDateObj() || new Date(),
+			showingMonth: this.selectedDateObj || new Date(),
 		};
 	},
 
 	computed: {
 		monthName() {
-			return this.viewingDate.toLocaleString(this.locale, { month: 'long', year: 'numeric' });
+			return this.showingMonth.toLocaleString(this.locale, { month: 'long', year: 'numeric' });
 		},
 
 		weekdays() {
-			const date = new Date();
-			const firstDOW = startOfWeek(date);
-			const weekdays = [...new Array(7)].map((_, i) => addDays(firstDOW, i));
-			return weekdays.map((d) => d.toLocaleDateString(this.locale, { weekday: 'short' }));
+			const firstDayOfWeek = startOfWeek(new Date());
+			const weekdays = Array.from({ length: 7 }, (_, i) => addDays(firstDayOfWeek, i));
+			return weekdays.map((date) => date.toLocaleDateString(this.locale, { weekday: 'short' }));
```

Comment:
In the Website version, designer Hannah Atkin wanted the English version of the calendar to display single-character headings (`S M T W T F S`) via:

```js
if (localizedDays.toString() === 'Sun,Mon,Tue,Wed,Thu,Fri,Sat') {
	return localizedDays.map(d => d.slice(0, 2));
}
```

Do we not want that anymore?

Reply:
Nope